examples/simultaneous_translation/docs/server_api.md

```diff
@@ -0,0 +1,173 @@
+# Introduction to the evaluation interface
+
+## **Server RESTful API**
+
+Assuming the server address is *server_url*
+
+---
+
+## Get number of sentences in test set
+
+* **Method:** : `GET`
+  
+*  **Endpoint** : *server_url*
+  
+*  **URL Params:**
+ 
+* **Success Response:**
+    
+    ```
+    {
+        "num_sentences": N
+    }
+    ``` 
+
+
+Where `N` is the number of sentences in test set
+
+---
+## Start a new evaluation session
+* **Method:** : `POST`
+  
+*  **Endpoint** : *server_url*
+  
+*  **URL Params:** 
+
+Should be used everytime starting to evaluate a new model.
+
+___
+## Obtain a source word from server
+
+* **Method:** : `GET`
+  
+*  **Endpoint** : *server_url*/src
+  
+*  **URL Params:** 
+  
+    ```
+    {
+        "sent_id": i
+    }
+    ``` 
+ 
+* **Success Response:**
+    
+    ```
+    {
+        "sent_id": i,
+        "segment_id": segment_id,
+        "segment": word
+    }
+    ```
+
+A source word in sentence *i* will be received from server. 
+Notice that the word is not tokenized. 
+```segment_id``` is the position of the word in sentence, starting from zero.
+The server will automatically move the pointer to next word. 
+When the pointer reach the end of sentence, "<\/s>" will be sent. 
+For example, for sentence "A B C". 
+The first time the request happens, "A" will be received,  the second time will be "B", the three time will be "C",  and after then will be always "<\/s>". The request should be made everytime the model decides to read a new word.
```

Comment:
```suggestion
The first time the request happens, "A" will be received,  the second time will be "B", the third time will be "C",  and after that, it will be always "<\/s>". The request should be made everytime the model decides to read a new word.
```